src/libaktualizr/uptane/fetcher_test.cc

```diff
@@ -107,8 +107,16 @@ TEST(fetcher, fetch_with_pause) {
   EXPECT_GE((finish - start), std::chrono::seconds(2));
 }
 
-/* Resume download interrupted by restart. */
-TEST(fetcher, fetch_restore) {
+TEST(fetcher, test_pause_ostree) {
+  Json::Value target_json;
+  target_json["hashes"]["sha256"] = "16ef2f2629dc9263fdf3c0f032563a2d757623bbc11cf99df25c3c3f258dccbe";
+  target_json["custom"]["targetFormat"] = "OSTREE";
+  target_json["length"] = 0;
+  Uptane::Target target("pause", target_json);
+  test_pause(target);
+}
```

Comment:
Can we also have `test_pause_binary`, which would basically be the same as how this used to be, but now reusing the `test_pause` function?

Reply:
Hm, strange. I made this function exactly  to use in both tests.

Reply:
I figured that was your intention, and it makes perfect sense to me!